ui/src/components/DocSearch/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { func } from 'prop-types';
+import docsearch from 'docsearch.js/dist/cdn/docsearch';
+import 'docsearch.js/dist/cdn/docsearch.css';
+import Search from '../Search';
+
+export default class DocSearch extends Component {
+  static propTypes = {
+    onSubmit: func.isRequired,
+  };
+
+  componentDidMount() {
+    docsearch({
+      apiKey: 'c7d22dbe731b27772278220687a87208',
+      // Used for identification when using Algolia's API
```

Comment:
Is it OK to make this information public?  If this is all generated locally, how is the API involved?  Could someone else use this key and ID to break our search?  Or possibly to inject malicious code into our users' browser sessions?